### PR TITLE
Issue 347

### PR DESCRIPTION
**Pull request by hurbes**

### CODE REVIEW

File: packages/stacked_generator/lib/src/generators/getit/stacked_locator_content_generator.dart [hunk 1]
```diff
@@ -22,8 +22,9 @@ class StackedLocatorContentGenerator extends BaseGenerator {
         .any((service) => service.type == DependencyType.PresolvedSingleton);
 
     writeLine(
-        '${hasPresolve ? 'Future' : 'void'} setupLocator() ${hasPresolve ? 'async' : ''} {');
-
+        '${hasPresolve ? 'Future' : 'void'} setupLocator({String? environment , EnvironmentFilter? environmentFilter}) ${hasPresolve ? 'async' : ''} {');
+    writeLine(
+        'final stacked = StackedLocator(environment: environment, environmentFilter: environmentFilter);');
```
Comment (FilledStacks) [inline packages/stacked_generator/lib/src/generators/getit/stacked_locator_content_generator.dart]: Why has the name changed to stacked here? 

Most apps use the term locator. This is going to cause a breaking change.

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Double checking that the locator name change is not going to be a breaking change for apps that use the stacked_generator

File: packages/stacked_generator/lib/src/generators/getit/stacked_locator_content_generator.dart (hunk 1 above)
Comment (hurbes) [reply packages/stacked_generator/lib/src/generators/getit/stacked_locator_content_generator.dart]: It's just the local variable within the `setupLocator` to register the dependencies with, the global variable stays the same as `locator`.

```dart
  writeLine('final locator = StackedLocator.instance;');
 ```

**Comment by hurbes** [comment]
I changed the implementation a bit and moved all the `Environment` related stuff from the constructor to its own function.

**Review by FilledStacks — APPROVED** [review]

**Pull request merged**